frontend/amundsen_application/api/quality/v0.py

```diff
@@ -0,0 +1,59 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+import logging
+from pkg_resources import iter_entry_points
+
+from http import HTTPStatus
+
+from flask import Response, jsonify, make_response, request, current_app as app
+from flask.blueprints import Blueprint
+from marshmallow import ValidationError
+from werkzeug.utils import import_string
+
+LOGGER = logging.getLogger(__name__)
+QUALITY_CLIENT_CLASS = None
+QUALITY_CLIENT_INSTANCE = None
+
+for entry_point in iter_entry_points(group='quality_client', name='quality_client_class'):
+    quality_client_class = entry_point.load()
+    if quality_client_class is not None:
+        QUALITY_CLIENT_CLASS = quality_client_class
+
+quality_blueprint = Blueprint('quality', __name__, url_prefix='/api/quality/v0')
+
+
+@quality_blueprint.route('/table', methods=['GET'])
+def get_table_quality_checks() -> Response:
```

Comment:
I'm wondering whether, in the spirit of making this extendable, we should call this route and method something like "/minimal_checks". Let's talk more about this IRL.